src/Data/Food/Recipe.elm

```diff
@@ -649,6 +649,7 @@ toStepsImpacts trigram results =
     , packaging = getImpact results.packaging
     , transform = getImpact results.recipe.transform
     , transports = getImpact results.transports.impacts
+    , trims = Just Unit.noImpacts
```

Comment:
What is the reason for using `Just Unit.noImpacts` instead of `Nothing` here?

Reply:
Good catch!